frontend/src/modules/history/components/Translation.js

```diff
@@ -206,6 +206,12 @@ export class TranslationBase extends React.Component<InternalProps, State> {
         );
 
         let className = 'translation ' + this.getStatus();
+
+        if(!user.isAuthenticated || isReadOnlyEditor) {
+            // This user can not copy translate
+            className += ' can-not-copy';
```

Comment:
The proper form would be `cannot` instead of `can-not`.

Reply:
Done